operator/internal/manifests/storage/options.go

```diff
@@ -12,6 +12,7 @@ type Options struct {
 	GCS         *GCSStorageConfig
 	S3          *S3StorageConfig
 	Swift       *SwiftStorageConfig
+	CAName      string
```

Comment:
I would err to keep this as is, because new storage types are inflight and they are some non-cloud-provider ones. I believe @eranra told me that IBM is looking after contributing one for their container object storage solution.